sonar-csharp-plugin/src/test/java/org/sonar/plugins/csharp/CSharpPluginTest.java

```diff
@@ -33,7 +33,7 @@
 import org.sonarsource.dotnet.shared.plugins.CodeCoverageProvider;
 import org.sonarsource.dotnet.shared.plugins.DotNetSensor;
 import org.sonarsource.dotnet.shared.plugins.EncodingPerFile;
-import org.sonarsource.dotnet.shared.plugins.FileCacheSensor;
+import org.sonarsource.dotnet.shared.plugins.AbstractFileCacheSensor;
```

Comment:
This line looks weird. It adds `Abstract` variant while the class uses `CSharp` variant. Is that intentional?

Reply:
It wasn't used